.pipelines/pipeline.yml

```diff
@@ -0,0 +1,103 @@
+pr:
+- master
+
+trigger:
+  branches:
+    include:
+    - master
+
+jobs:
+- job: Test
+  pool:
+    vmImage: 'Ubuntu-16.04'
+
+  variables:
+    GOBIN:  '$(GOPATH)/bin' # Go binaries path
+    GOROOT: '/usr/local/go1.11' # Go installation path 
```

Comment:
use go1.12?

Reply:
sure, just roll with the latest installed on runner, maintained here: https://github.com/Microsoft/azure-pipelines-image-generation/blob/master/images/linux/scripts/installers/go.sh